runtime/appruntime/apisdk/api/encore_routes.go

```diff
@@ -12,6 +12,7 @@ import (
 func (s *Server) registerEncoreRoutes() {
 	s.encore.HandlerFunc(wildcardMethod, "/healthz", s.handleHealthz)
 	s.encore.Handle("POST", "/pubsub/push/:subscription_id", s.handlePubsubPush)
+	s.encore.Handle("POST", "/auth_handler", s.handleRemoteAuthCall)
```

Comment:
Can we change the URL to `/authhandler` instead? Underscores in URLs are a bit unusual I think